catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DumpCommand.java

```diff
@@ -81,11 +92,13 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(DumpCommand.class);
 
-  private static final String ZIP_COMPRESSION = "zipCompression";
-
   private List<MetacardTransformer> transformers = null;
 
-  @VisibleForTesting volatile Optional<QueryResponseTransformer> zipCompression = Optional.empty();
+  private static final String METACARD_PATH = "metacards" + File.separator;
+
+  private static final String CONTENT = "content";
+
+  private static final int BUFFER_SIZE = 4096;
```

Comment:
❗️ Since we're doing each metacard serially, I would crank this buffer size up considerably larger. Since we're only serializing the metadata w/r/t this buffer, doing something like 10MB is fine. This way nearly every metacard will fit within the buffer and we wont pay the cost of going to disk to write out that stream, which could be a huge performance hit.

Reply:
This is a good idea. I changed this to 10000000 bytes.